README.md

```diff
@@ -63,6 +63,11 @@ Total: 10, Dialing: 1, Established: 9, Closed: 0, Error: 0, NotInitiated: 0
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
 --- myhttpsamplehost.com:80 tcp test statistics ---
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
+--- Summary of Established connections ---
+Total established connections: 10
+Max concurrent established connections: 10
+Last number of established connections: 10
```

Comment:
I suggest to rewrite as "Number of established connections on closure"

Reply:
👍